README.md

```diff
@@ -1,24 +1,57 @@
-# README
+# Budget-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- Is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby on Rails
 
-* System dependencies
+### Video Link
+- Here is a link to the video explaining the project [link](https://www.loom.com/share/bcb5bbd8f88848afb995e0b694c26583)
 
-* Configuration
+## Live Demo
 
-* Database creation
+No live demo available
 
-* Database initialization
+### Download it as a ZIP file
 
-* How to run the test suite
+or
 
-* Services (job queues, cache servers, search engines, etc.)
+### Clone it in your local machine using GIT
+```
+Clone repo: https://github.com/kwekubright/expenses_app.git
+```
 
-* Deployment instructions
+then
 
-* ...
+```run
+
+- bundle install
+- rails db:migrate
+- rails server
+
+```
+### Live
+Link [link](https://6353a7ef5b4a5e5f60b2452d--beautiful-rugelach-0b2089.netlify.app/)
+
+## Authors
+
+👤 **Bright Ahiadeke**
+
+- GitHub: [@kwekubright](https://github.com/kwekubright)
+- Twitter: [@kwekubright_](https://twitter.com/kwekubright_)
+- LinkedIn: [Bright ahiadeke](https://www.linkedin.com/in/kwekubright/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/kwekubright/expenses_app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
Please make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).